rewards/boost/boost_utils.py

```diff
@@ -1,10 +1,12 @@
+from toolz.itertoolz import cons
```

Comment:
what does this do? Don't see it referenced in the code

Reply:
it was left there, feel free to remove